internal/pkg/diff/reporter.go

```diff
@@ -200,8 +200,9 @@ func (r *Reporter) objectPath(value reflect.Value) string {
 
 func valuesDiff(remote, local reflect.Value) []string {
 	var out []string
+	includeType := remote.IsValid() && local.IsValid() && remote.Type().String() != local.Type().String()
```

Comment:
Do `diff` som pridal, nech vypise typ, ak sa lisi.
Pozeral som na to dost dlho, ze preco mi `diff` pada na rovnakych hodnotach.